Integrations/integration-ServiceNow.yml

```diff
@@ -1360,4 +1398,4 @@ script:
     description: Upload new file
   isfetch: true
 hidden: false
-releaseNotes: "Support templates on ticket/incident creation"
\ No newline at end of file
+releaseNotes: Added support for custom fields in commands: create and update.
```

Comment:
I am not sure that is is good to change the names now because playbooks already use this names.